pkg/config/dynamicconfig.go

```diff
@@ -195,13 +195,21 @@ func (cd *ConfigData) updateCM(old, cur interface{}) {
 		return
 	}
 	// if data has not changed then dont load configmap
-	reconcilePolicyReport, updateWebook := cd.load(*cm)
-	if reconcilePolicyReport {
+	reconcilePolicyReport, updateWebhook := cd.load(*cm)
+
+	cd.ReconcilePolicyReport(reconcilePolicyReport)
+	cd.UpdateWebhook(updateWebhook)
+}
+
+func (cd *ConfigData) ReconcilePolicyReport(reconcile bool) {
+	if reconcile {
 		cd.log.Info("resource filters changed, sending reconcile signal to the policy controller")
 		cd.reconcilePolicyReport <- true
 	}
+}
 
-	if updateWebook {
+func (cd *ConfigData) UpdateWebhook(update bool) {
```

Comment:
Do we need to extract this into a separate method as it's just a `send` request to the channel? And the reasons for updating the webhook are different, this log message could not reflect its purpose.

Reply:
The `updateWebhookConfigurations` channel is currently private, lies in the `config` package, and isn't directly accessible in `monitor.go` which lies in `webhookconfig` package, so this function is needed to make any changes in the webhook configurations through the webhook monitor. We can also make the channel public, but I'd need your feedback on its acceptability.